### PR TITLE
Set Pathfinding API request and get a response

### PR DESCRIPTION
**Pull request by Dominik1999**
This is about Issue #82 and sets the Pathfinding API to get information as a customer

### CODE REVIEW

**Comment by LefterisJP** [comment]
Guys please enforce the contributing guidelines also in the raiden services repo: https://github.com/raiden-network/raiden/blob/d05bb74d70fa196bf14bf334546014e7316fb8f2/CONTRIBUTING.md

Check the commit message guide

File: pathfinder/api/rest.py [hunk 1]
```diff
@@ -107,6 +108,14 @@ def get(self, token_network_address: str):
...
         return {'result': paths}, 200
 
+class InfoResource(PathfinderResource):
+    def get(self):
+        ip = socket.gethostbyname(socket.gethostname()) ## this is the internal IP address, which is a feature that will be scoped out soon anyway
+        settings = 'PLACEHOLDER'
+        version = '0.1'
+        dude_who_runs_it = 'Dominik'
+
+        return {'IP': ip, 'Settings': settings, 'Version': version, 'Dude who runs it': dude_who_runs_it}, 200
```
Comment (palango) [inline pathfinder/api/rest.py]: Let's see what stuff we need here, as you wrote IP doesn't make to much sense.

`Dude who runs it` should probably be `operator`.

You should be able to get the version from the package info, see how raiden does it: https://github.com/raiden-network/raiden/blob/45c4c47ec08e1e232b5f0b24386c8fc0a46f647a/raiden/utils/__init__.py#L173

And for me, maybe a `message` field with some info that would be displayed in the Raiden client might make sense as well.

File: pathfinder/api/rest.py [hunk 2]
```diff
@@ -127,3 +137,5 @@ def __init__(self, pathfinding_service: PathfindingService) -> None:
     def run(self, port: int = API_DEFAULT_PORT):
         self.rest_server = WSGIServer((API_HOST, port), self.flask_app)
         self.server_greenlet = gevent.spawn(self.rest_server.serve_forever)
+
```
[the remark this answers is not in the document]
Comment (Dominik1999) [reply pathfinder/api/rest.py]: Are you sure? When I comment these lines out I get an error 
`E       AttributeError: 'ServiceApi' object has no attribute 'run'`

[the remark this answers is not in the document]
Comment (Dominik1999) [reply pathfinder/api/rest.py]: This is from the class ServiceApi which I did not create

File: pathfinder/api/rest.py [hunk 3]
```diff
@@ -107,6 +108,14 @@ def get(self, token_network_address: str):
 
         return {'result': paths}, 200
 
+class InfoResource(PathfinderResource):
+    def get(self):
+        ip = socket.gethostbyname(socket.gethostname()) ## this is the internal IP address, which is a feature that will be scoped out soon anyway
```
Comment (LefterisJP) [inline pathfinder/api/rest.py]: also please run flake8 on PFS and MS. This comment line will hit on flake8. Needs an additional space before #

Comment (palango) [reply pathfinder/api/rest.py]: Yes, there is https://github.com/raiden-network/raiden-pathfinding-service/issues/88

No need to do this in this PR

**Comment by palango** [comment]
There's a fix for the build problem in master now, you just need to rebase.

**Comment by palango** [comment]
Superseeded by #95

**Pull request closed without merging**